src/test/regress/Makefile

```diff
@@ -47,6 +47,8 @@ check-worker: all
 
 check-multi: all tempinstall-main
 	$(pg_regress_multi_check) --load-extension=citus \
+	--server-option=citus.task_tracker_delay=10ms \
+	--server-option=citus.remote_task_check_interval=5ms \
```

Comment:
Can we set remote_task_check_interva to smaller numbers such as 1ms. I have seen extra performance on my local tests with 1ms. Is there any blocker for this?